src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/lib/__tests__/ensure_minimum_time.test.js

```diff
@@ -0,0 +1,28 @@
+import { ensureMinimumTime } from '../ensure_minimum_time';
+
+describe('ensureMinimumTime', () => {
+  it('resolves single promise', async (done) => {
+    const promiseA = new Promise(resolve => resolve('a'));
+    const a = await ensureMinimumTime(promiseA, 0);
+    expect(a).toBe('a');
+    done();
+  });
+
+  it('resolves multiple promises', async (done) => {
+    const promiseA = new Promise(resolve => resolve('a'));
+    const promiseB = new Promise(resolve => resolve('b'));
+    const [ a, b ] = await ensureMinimumTime([promiseA, promiseB], 0);
+    expect(a).toBe('a');
+    expect(b).toBe('b');
+    done();
+  });
+
+  it('resolves in the amount of time provided, at minimum', async (done) => {
+    const startTime = new Date().getTime();
+    const promise = new Promise(resolve => resolve());
+    await ensureMinimumTime(promise, 100);
+    const endTime = new Date().getTime();
+    expect(endTime - startTime).toBeGreaterThanOrEqual(100);
```

Comment:
I don't really know why yet, but I see this sometimes with this test:
```
● ensureMinimumTime › resolves in the amount of time provided, at minimum

    expect(received).toBeGreaterThanOrEqual(expected)
Expected value to be greater than or equal:
      100
    Received:
      99
```

Reply:
It's possible it only happens in watch mode when running the jest tests but it's still weird...